src/makeRobotAccountant.js

```diff
@@ -4,9 +4,23 @@
  *
  * @return {function}
  */
+const ERROR_MESSAGE = 'Bzzz... Error!';
+const USAGE_LIMIT = 3;
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return a => {
+    count++;
+
+    const isUsageOverLimit = count > USAGE_LIMIT;
+    const isUsageCountEven = count % 2 === 0;
+    const isError = isUsageOverLimit && isUsageCountEven;
+
+    return b => isError
+      ? ERROR_MESSAGE
+      : a + b;
+  };
```

Comment:
Since we definitely expect a number in the arguments, it is possible to personalize the argument names a bit.
```suggestion
  return num1 => {
    operationsCount++;

    const isUsageOverLimit = operationsCount > USAGE_LIMIT;
    const isUsageCountEven = operationsCount % 2 === 0;
    const isError = isUsageOverLimit && isUsageCountEven;

    return num2 => isError
      ? ERROR_MESSAGE
      : num1 + num2;
  };
```